protobufs/livekit_models.proto

```diff
@@ -601,3 +601,60 @@ enum AudioTrackFeature {
   TF_ENHANCED_NOISE_CANCELLATION = 5;
 }
 
+message DataStream {
+
+  // enum for operation types (specific to TextStreamHeader)
```

Comment:
:nit: there is no message called `TextStreamHeader`. When it is written like that, it makes it look like a type and I would want to search for it to look at the shape of it.